src/main/java/seedu/club/model/group/Group.java

```diff
@@ -3,13 +3,14 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.club.commons.util.AppUtil.checkArgument;
 
+//@@author yash-chowdhary
 /**
  * Represents a member's Group in the club book
  * Guarantees: immutable; is valid as declared in {@link #isValidGroup(String)}
  */
 public class Group {
     public static final String MESSAGE_GROUP_CONSTRAINTS =
-            "member groups should only contain alphanumeric characters, and it should not be blank";
+            "Member groups should only contain alphanumeric characters, and it should not be blank";
```

Comment:
Perhaps just "Groups" instead of "Member groups" sounds more intuitive since group only means one thing: group of a person.

Reply:
Ok, I'll update that. Thanks!